calpip/Task1/Task 1-1.txt

```diff
@@ -1,21 +1,21 @@
 Dear Sir or Madam,
 
-My name is Wellington and I work voluntarily in a community organization that 
-makes a wonderful job for local children. I am writing to ask your company for
+My name is Wellington and I volunteer for a community organization that 
+provides a wonderful service for local children. I am writing to ask your company for
```

Comment:
`makes a wonderful job` sounds awkward, you can say `does a wonderful job` to use that ("a good job") phrasing correctly, but it still sounds awkward

I thought `provides a wonderful service` sounds the most natural